00-HTML-CSS-basics/Exercise 2.5.2.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        <!-- Place favicon.ico in the root directory -->
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/grid.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="showcase"><b>Showcase</b></div>
+            <div class="info">Exercise 2.5.2. This exercise is about learning how to desing a site with css layouts. 
```

Comment:
This should be on an extra line. The text has more than one line so it should be wrapped this way:

```
<div>
    My text
    Goes here
</div>
```